### ISSUE TITLE
Migration fails running multiple instances on empty mysql databases.

### ISSUE THREAD

**Issue by mhaley-miovision** [flyway/flyway#1096]
When running a migration in parallel with multiple nodes the migration will fail in two ways.

The first is if there is no schema_version table in the database, one node will get a migration exception that it is unable to create the table.  A workaround is to create an empty schema_version table.

If the schema_version table already exists, but is empty, the MySQLTable.doLock function fails to properly lock the table.  A workaround here is to insert a fake row into the schema_version table before starting the parallel migration.

How I reproduced these issues:
1. Create a new mysql database
2. Create multiple migration scripts (I used our production scripts, so am unable to post them here.), with the first migration being a create table script (ours created 6 tables).
3. Run migrations in parallel.  I did this by launching several different spring applications in a for loop, launching each app start function in its own thread.

**Comment by axelfontaine** [comment]
Duplicate of #1067
